.github/actions/setup/action.yml

```diff
@@ -7,30 +7,23 @@ runs:
       uses: pnpm/action-setup@v2
       with:
         version: 8.6.5
-
     - name: Setup node 16.x
       uses: actions/setup-node@v3
       with:
         node-version-file: .nvmrc
         registry-url: https://registry.npmjs.org
         cache: pnpm
 
-    - name: Get foundry version
-      id: foundry-version
-      run: echo ::set-output name=version::$(cat .foundryrc)
-
     - name: Setup foundry
       uses: foundry-rs/foundry-toolchain@v1
-      with:
-        version: $${ steps.foundry-version.outputs.version }
 
     - name: Install node dependencies
       shell: bash
       run: pnpm install --frozen-lockfile
-
     - name: Derive appropriate SHAs for base and head for `nx affected` commands
       uses: nrwl/nx-set-shas@v3
-
+      with: 
+        main-branch-name: "develop"
```

Comment:
It defaults to main without this. Never noticed that since all my repos and lattice use main